src/peergos/shared/user/fs/SplitFragmenter.java

```diff
@@ -12,11 +12,18 @@ public double storageIncreaseFactor() {
     }
 
     public byte[][] split(byte[] input) {
+        //calculate padding length to align to 256 bytes
+        int padding = 0;
+        int mod = input.length % 256;
+        if (mod == 0 && input.length > 0)
+            padding = 256 - mod;
+        //align to 256 bytes
```

Comment:
doesn't this mean padding is always 0, unless length is already aligned, when it is 256?